src/main/java/core/basesyntax/GetArray.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public interface GetArray {
+    default Figure[] getArray() {
+        int length = new Random().nextInt(10);
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] rez = new Figure[length];

```

Comment:
what about informative variable names
```suggestion
        Figure[] figures = new Figure[length];

```